src/sinks/influxdb/logs.rs

```diff
@@ -174,14 +166,16 @@ struct InfluxDbLogsEncoder {
     protocol_version: ProtocolVersion,
     measurement: String,
     tags: HashSet<String>,
-    encoding: EncodingConfig<Encoding>,
+    transformer: Transformer,
 }
 
 impl HttpEventEncoder<BytesMut> for InfluxDbLogsEncoder {
     fn encode_event(&mut self, event: Event) -> Option<BytesMut> {
         let mut event = event.into_log();
         event.insert("metric_type", "logs".to_string());
-        self.encoding.apply_rules(&mut event);
+        let mut event = Event::from(event);
+        self.transformer.transform(&mut event);
+        let mut event = event.into_log();
```

Comment:
```suggestion
        let mut event = {
          let mut event = Event::from(event);
          self.transformer.transform(&mut event);
          event.into_log()
      }
```

Maybe only marginally more readable. I just found what is here with the reassignment to `event` a bit hard to follow.

Reply:
Did a change in the same spirit: 3d4d34dc215cf92a3e29668729126ccb641ef3da